tests/unit/tasks/serve-test.js

```diff
@@ -73,23 +70,23 @@ describe('serve task', function() {
       );
     });
 
-    it(`Serves ember app from given path`, function() {
+    it(`Serves ember app from given path`, async function() {
       runServeTask('docs');
-      return RSVP.resolve().then(() => {
-        expect(ui.output).to.be.contains('– Serving on');
-      });
+
+      await Promise.resolve();
```

Comment:
that looks like a very strange test to me, but it already was like that before, so I guess that's fine?! 😅